src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php

```diff
@@ -103,16 +105,14 @@ public function process(ContainerBuilder $container): void
                 $definition->addMethodCall('setHidden', [true]);
             }
 
-            if (!$description) {
-                if (!$r = $container->getReflectionClass($class)) {
-                    throw new InvalidArgumentException(\sprintf('Class "%s" used for service "%s" cannot be found.', $class, $id));
-                }
-                if (!$r->isSubclassOf(Command::class)) {
-                    throw new InvalidArgumentException(\sprintf('The service "%s" tagged "%s" must be a subclass of "%s".', $id, 'console.command', Command::class));
-                }
```

Comment:
[optimization] the reflection variable and exceptions have already been defined earlier